buffer/src/main/java/io/netty5/buffer/api/Buffer.java

```diff
@@ -399,6 +400,70 @@ default Buffer writeBytes(byte[] source, int srcPos, int length) {
         return this;
     }
 
+    /**
+     * Writes into this buffer from the source {@link ByteBuffer}.
+     * This updates the {@linkplain #writerOffset() write offset} of this buffer and also the position of
+     * the source {@link ByteBuffer}.
+     *
```

Comment:
We can also call out that the behaviour is undefined if the given `ByteBuffer` is an alias of this `Buffer`.